test.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+set -e
+clean_up () {
+    ARG=$?
+
+    if [[ $ARG -ne 0 ]]; then
+      echo "Test failed, showing docker logs"
+      echo "- Mockserver"
+      docker-compose -f ./test/docker-compose.yml logs mockserver
+      echo "- Fluent Bit"
+      docker-compose -f ./test/docker-compose.yml logs newrelic-fluent-bit-output
+    fi
+
+    echo "Cleaning up"
+    rm -r ./test/testdata || true
+    docker-compose -f ./test/docker-compose.yml down
+
+    exit $ARG
+}
+trap clean_up EXIT
+
+function check_logs {
+  curl -X PUT -s --fail "http://localhost:1080/mockserver/verify" -d @test/verification.json >> /dev/null
+  RESULT=$?
+  return $RESULT
+}
+
+function check_mockserver {
+  curl -X PUT -s --fail "http://localhost:1080/mockserver/status" >> /dev/null
+  RESULT=$?
+  return $RESULT
+}
+
+# Create testdata folder and log file
+mkdir ./test/testdata || true
+touch ./test/testdata/fbtest.log
+
+# Initialize
+if [ ${CI:-no} = "no" ]; then
```

Comment:
Could You add some description about `CI` env var

Reply:
Sure! Thanks!!